compiler/codegen/OMRAheadOfTimeCompile.cpp

```diff
@@ -115,6 +116,26 @@ void OMR::AheadOfTimeCompile::traceRelocationOffsets(uint8_t *&cursor, int32_t o
       }
    }
 
+
+uint8_t* OMR::AheadOfTimeCompile::initializeAOTRelocationHeader(TR::IteratedExternalRelocation *relocation)
+   {
+   TR_UNIMPLEMENTED();
+   return 0;
+   }
+
+
+void OMR::AheadOfTimeCompile::processRelocations()
+   {
+   TR_UNIMPLEMENTED();
+   }
+
+uint8_t *
+OMR::AheadOfTimeCompile::initializeCommonOMRAOTRelocationHeader(TR::IteratedExternalRelocation *relocation, TR::RelocationRecord *reloRecord)
```

Comment:
"OMRAOT" seems redundant in this function name.

Reply:
Hello @mstoodle, thank you for comment. This name is used to avoid overriding this method with openJ9. The problem was in when I was building with autoconf and cmake, one of those build systems was complaining. Maybe an alternative name should be used, can you suggest one?

Reply:
maybe just `initializeCommonRelocationHeader` since it's already in the `OMR` namespace and the `AheadOfTimeCompile` class ?

Reply:
Thank you for your suggestion @mstoodle. The latest force-push has changing addressing your suggestions.